webui/react/Makefile

```diff
@@ -8,9 +8,17 @@ get-deps:
 build:
 	npm run build
 
+.PHONY: build-storybook
+build-storybook:
+	npm run build-storybook
+
 .PHONY: clean
-clean:
-	rm -rf build node_modules/
+clean: clean-deps
+	rm -rf build build-storybook
+
+.PHONY: clean-deps
+clean-deps:
+	rm -rf node_modules/
```

Comment:
thought: I don't have strong opinions here, but just thought this might be overkill vs just a `rm -rf build build-storybook node_modules/`. This is the only place with `clean-deps` target in the project, but not opposed to the idea.

Reply:
I think https://github.com/determined-ai/determined/pull/445 will do this better so I'll just drop it here.